### DISCUSSION TITLE
Which wheels can we avoid re-inventing?

### DISCUSSION THREAD

**Discussion by Alex-Yates** [dataplat/DataSaturdays#165] [Q&A]
One of the challenges with sqlsaturday is that a lot of the tools that were genuinely helpful early on were eventually overtaken by other services like sessionise.

At the time of writing, the following are listed as important assetts in the readme: 

1. Administrative tools for event organizers to use when building their site
2. Rating of sessions and events

I wonder whether these need to be centralised and maintained in house in 2021+, or whether we are creating work, overhead and tech debt for ourselves for minimal return.

Perhaps point 1 is better served by a repo with guidance to organisers for self hosting their own wordpress site? Or perhaps we can publish a standard notnet core web app on github with instructions to fork the repo and self host? Perhaps we can talk to Microsoft and ask for free hosting in Azure for these sites, since I'm assuming we are still going to be primarily a microsofty/azure heavy crowd?

As for point 2, can we really expect to build something that's more useful and easier to maintain than sessionise?

Rather than getting into the weeds on these specific points, the emphasis of my question is whether we can be as deliberate about what we DON'T do, as we are about what we DO do. I feel this would help us to avoid creating another powerful centralised org/admin team and support us to keep as much autonomy as possible with the event owners themselves.

**Comment by way0utwest** [comment]
I think this is too general a note for discussion of specifics, but it is a good starting point for the scope of tooling/integrations. I am adding a new items for
- speakers (#18 )
- Ratings  (#19 )
- Admin tools #20
